src/index.html

```diff
@@ -1,13 +1,709 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Met Museum</title>
+    <link rel="icon" href="./images/logo.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css">
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <nav class="header__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="logo"
+              >
+            </a>
+
+            <a
+              href="tel:+1 212-535-7710"
+              class="icon icon--phone"
+            >
+              <div class="icon__contacts">
+                The Met Fifth Avenue <br>
+                +1 212-535-7710 <br>
+                The Met Cloisters <br>
+                +1 212-923-3700
+              </div>
+            </a>
+
+            <a href="#menu" class=" icon icon--menu"></a>
```

Comment:
According to the mockup, the menu button icon changes on hover

![image](https://user-images.githubusercontent.com/78492113/218106582-81bee162-949c-4de5-be74-0377a4364bcb.png)